back/engines/commercial/analytics/db/views/analytics_fact_project_statuses_v01.sql

```diff
@@ -0,0 +1,37 @@
+WITH regular_project_statuses AS -- project statuses that do not need to be calculated
+         (SELECT item_id as project_id, action as status, acted_at as timestamp
```

Comment:
we've now got a bit of a naming convention of using dimension_ for all the dimension columns

Reply:
so in the output view here we'd have dimension_project_id and dimension_data_id - id is needed on the date column so that rails can use it as a key